src/services/EdFi.DataManagementService.Api.Tests.E2E/Features/Resources/CreateResourcesValidation.feature.disabled

```diff
@@ -0,0 +1,184 @@
+                    }
+                  """
+             Then it should respond with 403
+              And the response body is
+                  """
+                    {
+                        "detail": "Data validation failed. See 'validationErrors' for details.",
+                        "type": "urn:ed-fi:api:bad-request:data",
+                        "title": "Data Validation Failed",
+                        "status": 400,
+                        "correlationId": null,
+                        "validationErrors": {
+                            "$.codeValue": [
+                                "CodeValue is required."
+                                ],
+                            "$.namespace": [
+                                "Namespace is required."
+                                ],
+                            "$.shortDescription": [
+                                "ShortDescription is required."
+                                ]
+                        }
+                    }
+                  """
+
+        @ignore
+        Scenario: Verify POST of existing record without changes
+             Give a POST request is made to "/ed-fi/absenceEventCategoryDescriptors" with
+                  """
+                    {
+                        "codeValue": "Sick Lave",
+                        "description": "Sick Leave",
+                        "namespace": "uri://ed-fi.org/AbsenceEventCategoryDescriptor",
+                        "shortDescription": "Sick Leave"
+                    }
+                  """
+             When a POST request is made to "/ed-fi/absenceEventCategoryDescriptors" with
+                  """
+                    {
+                        "codeValue": "Sick Lave",
+                        "description": "Sick Leave",
+                        "namespace": "uri://ed-fi.org/AbsenceEventCategoryDescriptor",
+                        "shortDescription": "Sick Leave"
+                    }
+                  """
+             Then it should respond with 200
+              And the record can be retrieved with a GET request
+
+        @ignore
+        Scenario: Verify POST of existing record (change non-key field) works
+             When a POST request is made to "/ed-fi/absenceEventCategoryDescriptors" with
+                  """
+                    {
+                        "codeValue": "Sick Lave",
+                        "description": "Sick Leave Edit",
+                        "namespace": "uri://ed-fi.org/AbsenceEventCategoryDescriptor",
+                        "shortDescription": "Sick Leave"
+                    }
+                  """
+             Then it should respond with 200
+
+        @ignore
+        Scenario: Verify error handling when resource ID is included in body on POST
+            # The id value should be replaced with the resource created in the Background section
+             When a POST request is made to "/ed-fi/absenceEventCategoryDescriptors/{id}" with
```

Comment:
Please remove `/{id}`. The critical part is the value on line 164 below, in the message body.

Reply:
Removed